_data/banking.yml

```diff
@@ -330,6 +330,14 @@ websites:
       software: Yes
       doc: https://www.firsttechfed.com/OnlineServices/bankonlinesecurely/SecurityKey
 
+    - name: FirstBank (Colorado)
```

Comment:
On the contrary, there are multiple other FirstBanks, including [this](https://www.firstbanks.com/), [this](https://www.firstbank.com/), [this](http://www.1firstbank.com/vi/en), and [this](https://www.firstbankofberne.com/) one. I can take care of that as well.